tup-cms/settings_custom.py

```diff
@@ -1,57 +1,31 @@
-# TACC CMS Per-Site Resources - Configuration
-
-# In a project-specific configuration file like this, edit existing configuration values from `cms-site-template:/taccsite_cms/default_secrets.py` that must change to suit this project.
-
-# Until this custom configuration is automatically applied, to use it one must copy the content of it and append it to `cms-site-template:/taccsite_cms/secrets.py`
-
-# TUP CONFIGURATION
-
-########################
-# DJANGO SETTINGS
-########################
-
-_LDAP_ENABLED = False
+# CUSTOM SETTINGS VALUES.
+# TACC WMA CMS SITE:
+# *.TUP.TACC.UTEXAS.EDU
 
 ########################
 # DJANGO CMS SETTINGS
 ########################
 
-_CMS_TEMPLATES = (
-    ('tup-cms/templates/fullwidth.html', 'Fullwidth')
+CMS_TEMPLATES = (
+    ('tup-cms/templates/fullwidth.html', 'Fullwidth'),
+    ('guide.html', 'Guide'),
+    ('guides/getting_started.html', 'Guide: Getting Started'),
+    ('guides/data_transfer.html', 'Guide: Data Transfer'),
+    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
+    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
 )
 
-########################
-# GOOGLE ANALYTICS
-########################
-
-_GOOGLE_ANALYTICS_PROPERTY_ID = "UA-96034853-2"
```

Comment:
@taoteg I noticed [you had asked about this property](https://github.com/TACC/Core-CMS-Resources/pull/74/commits/4044081bd516440efa5b7246bfd51ec53dd142ac#diff-09fb7241546cad246d7fbb69434ba551418ca43d62e9cd6eb1f3f25e9d860345R24).

It is the current ID used on https://www.tacc.utexas.edu (found by searching markup for `UA-`).
- I put it in assuming we'd not create a new property in Google Analytics.
- I do not have access to this property.